src/features/duplicates/store.tsx

```diff
@@ -2,51 +2,60 @@ import { ZetkinPerson } from 'utils/types/zetkin';
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { RemoteList, remoteList } from 'utils/storeUtils';
 
-export interface ZetkinDuplicate {
-  duplicatePersons: ZetkinPerson[];
+export interface PotentialDuplicate {
+  duplicates: ZetkinPerson[];
   dismissed: string | null;
   id: number;
   merged: string | null;
   organization: { id: number; title: string };
   status: 'merged' | 'dismissed' | 'pending';
 }
 
-export interface DuplicatesStoreSlice {
-  duplicatesList: RemoteList<ZetkinDuplicate>;
+export interface PotentialDuplicatesStoreSlice {
+  potentialDuplicatesList: RemoteList<PotentialDuplicate>;
 }
 
-const initialState: DuplicatesStoreSlice = {
-  duplicatesList: remoteList(),
+const initialState: PotentialDuplicatesStoreSlice = {
+  potentialDuplicatesList: remoteList(),
 };
 
-const duplicatesSlice = createSlice({
+const potentialDuplicatesSlice = createSlice({
   initialState,
-  name: 'duplicates',
+  name: 'potentialDuplicates',
   reducers: {
-    dismissedDuplicate: (state, action: PayloadAction<ZetkinDuplicate>) => {
-      const duplicate = action.payload;
-      const item = state.duplicatesList.items.find(
-        (item) => item.id === duplicate.id
+    dismissedPotentialDuplicate: (
+      state,
+      action: PayloadAction<PotentialDuplicate>
+    ) => {
+      const potentialDuplicate = action.payload;
+      const item = state.potentialDuplicatesList.items.find(
+        (item) => item.id === potentialDuplicate.id
       );
 
       if (item && item.data) {
-        item.data.dismissed = duplicate.dismissed;
-        item.data = { ...duplicate };
+        item.data.dismissed = potentialDuplicate.dismissed;
+        item.data = { ...potentialDuplicate };
```

Comment:
This is a little bit odd to me. I'm not sure exactly what `potentialDuplicate` will contain here, but I'm guessing it contains the full `PotentialDuplicate` item from the API. Whatever happens on line 37 vill no doubt overwrite the assignment from line 36.

If the API does indeed respond with the full item, this should be enough:

```suggestion
        item.data = potentialDuplicate;
```

Reply:
Looking at your commit c90777e8da9cd293b9c8f8b44b9abde614231257 I believe you misunderstood this comment. The problem is not that destructuring overwrites things. The point is that BECAUSE it overrides things, the first line is unnecessary. Note that I remove two lines and add one line in my proposed change above. I think you should do the the same.